python/hsfs/core/arrow_flight_client.py

```diff
@@ -566,8 +573,23 @@ def _serialize_featuregroup_connector(fg, query, on_demand_fg_aliases):
                     connector["filters"] = _serialize_filter_expression(
                         join_obj._query._filter, join_obj._query, True
                     )
+    elif fg.time_travel_format == "DELTA":
+        connector["time_travel_type"] = "delta"
+        connector["type"] = fg.storage_connector.type
+        connector["options"] = fg.storage_connector.connector_options()
```

Comment:
what if delta table is stored on hopsfs and not s3?

Reply:
no, it only work for s3